edn/src/query.rs

```diff
@@ -0,0 +1,1253 @@
+// Copyright 2016 Mozilla
```

Comment:
I would have expected this file to track via `git mv`… maybe replay that work to make sure we don't lose blame?

Reply:
I did this with a `Part 3b` that is just the file move.  I thought `git` would determine this automatically, but apparently it didn't.